docbook/src/docbkx/autoscale-gettingstarted.xml

```diff
@@ -399,7 +409,14 @@ xmlns:html="http://www.w3.org/1999/xhtml" xml:id="autoscale-getting-started" xml
                     <para>&PROD; does not configure any information within a server. You must configure your
                         services to function when each server is started. We recommend automating your
                         servers' startup processes with
-                        <link xlink:href="http://www.opscode.com/chef/">Chef</link>or a similar tool.</para>
+                        <link xlink:href="http://www.opscode.com/chef/">Chef</link> or a similar tool.</para>
+                </note>
+            </para>
+            <para>
+                <note>
+                    <para>&PROD; can utilize all Rackspace Cloud Server API operations. For more details, see the 
```

Comment:
Same comment as above with regards to "all Rackspace Cloud Server API operations".